using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
 Deleting your mobile property in Experience Platform Launch does not automatically delete your Campaign Standard mobile app.
 {% endhint %}
 
+### Handling deeplinks included in Campaign In-App messages
+
+A deeplink can be added to in-app messages delivered from Adobe Campaign. The link can be a website URL such as https://www.adobe.com or an app link which can be used to direct the user to a specific area of your app.
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Handling in-app message deeplinks on Android
+
+Website URL's are handled without any further action by the app developer. If an in-app message is clicked through and contains a valid URL, the device's default web browser will redirect to the URL contained in the in-app notification payload's `url` key (for fullscreen and alert messages) or `adb_deeplink` key (for local and push notifications). App links present in the notification payload can be handled by setting up URL schemes in the app. For more information about setting URL schemes for Android, see [Create Deep Links to App Content](https://developer.android.com/training/app-links/deep-linking).
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### Handling alert or fullscreen notfication deeplinks on iOS
+
+The default behavior for opening URL's and app links on iOS is to use the Safari browser. This behavior is expected if the included deeplink is a web URL, but it does not allow the app to handle app links. To provide a better customer experience, the AEP SDK provides a URL handler for use with alert or fullscreen notification deeplinks.
```

Comment:
- In the 1st sentence, please replace "URL's" with "URLs" and reword as follows:

- Reword this section as follows:
"To open URLs and app links on iOS, by default, use the Safari browser. If the included deeplink is a web URL, this is an expected behavior, but Safari does not allow the app to handle app links. To provide a better customer experience, the Experience Platform SDK provides a URL handler that you can use with alert or fullscreen notification deeplinks.